addons/im_livechat/static/tests/discuss_patch_tests.js

```diff
@@ -33,7 +34,7 @@ QUnit.test("No reaction button", async (assert) => {
         anonymous_name: "Visitor 11",
         channel_type: "livechat",
         livechat_operator_id: pyEnv.currentPartnerId,
-        channel_partner_ids: [pyEnv.currentPartnerId, pyEnv.publicPartnerId],
+        channel_partner_ids: [pyEnv.currentPartnerId],
```

Comment:
should remove `channel_partner_ids` everywhere and replace by proper usage of  `channel_member_ids`

Reply:
Done in prep